src/equations/compressible_navier_stokes_1d.jl

```diff
@@ -124,8 +124,8 @@ end
 
 # TODO: parabolic
 # This is the flexibility a user should have to select the different gradient variable types
-# varnames(::typeof(cons2prim)   , ::CompressibleNavierStokesDiffusion1D) = ("v1", "v2", "T")
-# varnames(::typeof(cons2entropy), ::CompressibleNavierStokesDiffusion1D) = ("w2", "w3", "w4")
+# varnames(::typeof(cons2prim)   , ::CompressibleNavierStokesDiffusion1D) = ("rho", "v1", "T")
+# varnames(::typeof(cons2entropy), ::CompressibleNavierStokesDiffusion1D) = ("w1", "w2", "w3")
```

Comment:
I am still not a fan of this. I my eyes we should have a separate set of variables of size `nvars_grad` to avoid the computation of unnecessary gradients like d_rho or d_w1. This commented section of code was intended to reflect such a desire. So changing it to include the unnecessary gradient values goes against its spirit.